e2e-tests/tests/gr-ignore-annotations/01-assert.yaml

```diff
@@ -30,6 +30,12 @@ metadata:
   name: gr-ignore-annotations
 status:
   conditions:
+  - reason: Initializing
+    status: "True"
```

Comment:
ummm, how could both `Initializing` and `Ready` have `status: True`?

Reply:
Honestly not sure, I see that behavior in our other operators.

Reply:
if Ready is true, Initializing should be false